lib/tracker_api/resources/search_result_container.rb

```diff
@@ -0,0 +1,12 @@
+module TrackerApi
+  module Resources
+    class SearchResultContainer
```

Comment:
Question 1: Notice this is including the `Shared::Base`. These resources don't require IDs though, so are there any changes to be made to the top level Base? It seems strange to have Resources without IDs. Am I missing anything?

Observation: I didn't make this a `Shared` resource since there isn't any attributes that are shared between a top-level `SearchResultContainer` or `EpicsSearchResult` or `StoriesSearchResult`, and this allows endpoints to be made specifically for epic or story search.